test/models/role_test.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class RoleTest < ActiveSupport::TestCase
+  # test "the truth" do
```

Comment:
Are there any model tests we can add here?

Reply:
In the test/system/authentication_system_test.rb, the admin role is tested for the sidekiq role. After pundit is merged, the routes file will change and roles will be authenticated through pundit policies.